src/pages/Schedulers/Schedulers.css

```diff
@@ -0,0 +1,14 @@
+.Schedulers-Container {
+  margin: 10px;
+}
+.Scheduler-button {
+  min-width: 300px;
+  max-width: 300px;
+}
+.ant-tree-node-content-wrapper {
+  font-size: 16px;
+}
+
+.h-100 {
```

Comment:
Please rename this classname to something meaningful like
`.SchedulersPage-Parent-Container`

Reply:
Resolved